src/main/resources/InstructorSampleData.json

```diff
@@ -2720,7 +2723,1319 @@
+      "recipientSection": "Tutorial Group 1",
+      "responseDetails": {
+        "answers": [
+          100
+        ],
+        "questionType": "CONSTSUM"
+      }
+    },
+    "FS3.Q1.Benny.4": {
+      "feedbackSessionName": "Second team feedback session (point-based)",
+      "courseId": "demo.course",
+      "feedbackQuestionId": "1",
+      "giver": "benny.c.tmms@gmail.tmt",
+      "recipient": "emma.f.tmms@gmail.tmt",
+      "giverSection": "Tutorial Group 1",
+      "recipientSection": "Tutorial Group 1",
+      "responseDetails": {
+        "answers": [
+          120
+        ],
+        "questionType": "CONSTSUM"
+      }
+    },
+    "FS3.Q1.Danny.1": {
+      "feedbackSessionName": "Second team feedback session (point-based)",
+      "courseId": "demo.course",
+      "feedbackQuestionId": "1",
+      "giver": "danny.e.tmms@gmail.tmt",
+      "recipient": "alice.b.tmms@gmail.tmt",
+      "giverSection": "Tutorial Group 1",
+      "recipientSection": "Tutorial Group 1",
+      "responseDetails": {
+        "answers": [
+          100
+        ],
+        "questionType": "CONSTSUM"
+      }
+    },
+    "FS3.Q1.Danny.2": {
+      "feedbackSessionName": "Second team feedback session (point-based)",
+      "courseId": "demo.course",
+      "feedbackQuestionId": "1",
+      "giver": "danny.e.tmms@gmail.tmt",
+      "recipient": "benny.c.tmms@gmail.tmt",
+      "giverSection": "Tutorial Group 1",
+      "recipientSection": "Tutorial Group 1",
+      "responseDetails": {
+        "answers": [
+          100
+        ],
+        "questionType": "CONSTSUM"
+      }
+    },
+    "FS3.Q1.Danny.3": {
+      "feedbackSessionName": "Second team feedback session (point-based)",
+      "courseId": "demo.course",
+      "feedbackQuestionId": "1",
+      "giver": "danny.e.tmms@gmail.tmt",
+      "recipient": "danny.e.tmms@gmail.tmt",
+      "giverSection": "Tutorial Group 1",
+      "recipientSection": "Tutorial Group 1",
+      "responseDetails": {
+        "answers": [
+          100
+        ],
+        "questionType": "CONSTSUM"
+      }
+    },
+    "FS3.Q1.Danny.4": {
+      "feedbackSessionName": "Second team feedback session (point-based)",
+      "courseId": "demo.course",
+      "feedbackQuestionId": "1",
+      "giver": "danny.e.tmms@gmail.tmt",
+      "recipient": "emma.f.tmms@gmail.tmt",
+      "giverSection": "Tutorial Group 1",
+      "recipientSection": "Tutorial Group 1",
+      "responseDetails": {
+        "answers": [
+          100
+        ],
+        "questionType": "CONSTSUM"
+      }
+    },
+    "FS3.Q1.Emma.1": {
+      "feedbackSessionName": "Second team feedback session (point-based)",
+      "courseId": "demo.course",
+      "feedbackQuestionId": "1",
+      "giver": "emma.f.tmms@gmail.tmt",
+      "recipient": "alice.b.tmms@gmail.tmt",
+      "giverSection": "Tutorial Group 1",
+      "recipientSection": "Tutorial Group 1",
+      "responseDetails": {
+        "answers": [
+          10
+        ],
```

Comment:
I believe the scores need to add up to 400. The scores that Emma gave might not be possible (10,100,100,140). Francis also has the same issue.

Reply:
Fixed thanks for the spot, must have changed when I was moving stuff around.